website/announcements/templates/events/event.html

```diff
@@ -0,0 +1,18 @@
+{% extends "events/event.html" %}
```

Comment:
This makes the order of the apps important right? I think you should add a comment about that in the settings.py

Reply:
That's right. I'll add a note. Once all of the unnecessary dependencies between apps are resolved, there should be a topological ordering or apps.